test/inductor/test_torchinductor_opinfo.py

```diff
@@ -934,6 +935,131 @@ def wrapper_noop_set_seed(op, *args, **kwargs):
 }
 
 
+# Custom replacements for assertEquals, in cases where a difference in value
+# may not indicate correctness.
+
+
+def get_sort_argsort_assert_equal_fn(is_argsort, args, kwargs):
+    # The normal assert_equal_fn suffices for a stable sort
+    def normal_assert_fn(
+        test_case_inst, x, y, *, atol=None, rtol=None, equal_nan=True, exact_dtype=True
+    ):
+        test_case_inst.assertEqual(
+            x[0],
+            y[0],
+            atol=atol,
+            rtol=rtol,
+            equal_nan=equal_nan,
+            exact_dtype=exact_dtype,
+        )
+
+    if "stable" in kwargs.keys():
```

Comment:
nit: 

```if kwargs.get("stable", False)```

There is no need for nested if or to create a dict_iterator to test membership.
```
>>> keys = {i: i for i in range(10000)}
>>> import timeit
>>> timeit.timeit('5000 in keys.keys()', globals={'keys': keys}, number=1000000)
0.04573630099184811
>>> timeit.timeit('5000 in keys', globals={'keys': keys}, number=1000000)
0.024304463004227728
>>> 
```